docs/source/dataframe-groupby.rst

```diff
@@ -97,20 +97,20 @@ Client as default:
     client = Client('scheduler:8786', set_as_default=True)
 
 Alternatively, if you prefer to avoid defaults, you can configure the global
-shuffling method by using the ``dask.config.set(shuffle=...)`` command.
+shuffling method with the ``dataframe.shuffle.algorithm`` configuration option.
 This can be done globally:
 
 .. code-block:: python
 
-    dask.config.set(dataframe_shuffle_algorithm='p2p')
+    dask.config.set({"dataframe.shuffle.algorithm": "p2p"})
```

Comment:
Using `dataframe_shuffle_algorithm` doesn't actually work, it looks like double underscores are needed. 

```python
In [1]: import dask

In [2]: dask.config.get("dataframe.shuffle.algorithm")

In [3]: dask.config.set(dataframe_shuffle_algorithm='p2p')
Out[3]: <dask.config.set at 0x10353bb20>

In [4]: dask.config.get("dataframe.shuffle.algorithm")

In [5]: dask.config.set(dataframe__shuffle__algorithm='p2p')
Out[5]: <dask.config.set at 0x10353be20>

In [6]: dask.config.get("dataframe.shuffle.algorithm")
Out[6]: 'p2p'
```

I think we should just stick with dictionary inputs (though I realize using non-`dict` inputs in this section predates https://github.com/dask/dask/pull/9991). This is what most folks are used to I think.